### ISSUE TITLE
Correct Using - Preprocessor directive

### ISSUE THREAD

**Issue by lneumeier** [omecra/de.omecra.oedt.updatesite#11]
I have this code:
```
&IF "{&ExpToFile}" = "On" &THEN

 USING Progress.Lang.*.
 /* Import class for save. */
 USING SaveTo.* FROM ASSEMBLY.

 &IF DEFINED(Save2) > 0 &THEN
   USING SaveTo2.* FROM ASSEMBLY.
 &ENDIF

&ENDIF /* of SveTo */
```
After "Correct Using"
```
&IF "{&ExpToFile}" = "On" &THEN

 USING Save2.*.
USING Save2.*.
USING Progress.Lang.*.
 &ENDIF

&ENDIF /* of SaveTo */
```

**Comment by Ton-Pe** [comment]
I noticed a bug which is related to this issue.
When trying to add SonarLint annotation to using statements, saving the file removes the annotation.

I have code:
`{&_proparse_ prolint-nowarn(usingpkg)} `
`USING Progress.Lang.* FROM PROPATH.`

But after saving (correcting using) it is just
`USING Progress.Lang.* FROM PROPATH.`

OEDT version 9.3.5. Using "old using-engine".  But same problem with new engine.

**Comment by lneumeier** [comment]
I look into it. Does this happen on save in an editor too?

**Comment by Ton-Pe** [comment]
This happens in Oedt Class editor and in Oedt Windows editor, but not in Oedt Proecdure editor or in Open Edge ABL editors.

It happens even if all other selections are deactivated except "Execute on save"
![image](https://user-images.githubusercontent.com/25682299/95415907-2f39c380-093a-11eb-88dc-0e89c5a57331.png)

**Comment by lneumeier** [comment]
Fixed in 9.3.6.
